waspc/src/Wasp/Generator/SdkGenerator.hs

```diff
@@ -87,3 +90,9 @@ sdkRootDirInProjectRootDir = [reldir|sdk/wasp|]
 
 sdkTemplatesDirInTemplatesDir :: Path' (Rel TemplatesDir) (Dir SdkTemplatesDir)
 sdkTemplatesDirInTemplatesDir = [reldir|sdk|]
+
+-- todo(filip): figure out where this belongs
```

Comment:
Can I help with this somehow?

Reply:
Yes, of course.

Where would you put it?

Reply:
Hm I am kind of not sure what is this -> so we are installing deps for user's code, right? In the top-level node modules? Hmmmmm. This sounds like it is not even related to Generator maybe? Oh yeah this is a bit tricky to figure out where to put. I would say either in Wasp.Project somewhere, but you will need stuff from Generator right, so that makes it a no-no mostly, we don't really to import from Generator there mostly.
Although we could argue that `runNodeCommandAsJob` maybe should go outside of Generator, if it is used by other parts of Wasp also, and doesn't depend on Generator! So in that case it makes sense.
Or, we just stick to Generator for now, that is not terrible, but we put this somewhere at its high level. Maybe create a new module in it called WaspProject or maybe put it in Common or some other module that might make sense. This feels a bit like hacking it, but is survivable I think.

Reply:
To whoever's doing this...

Please don't touch this for now. I'll handle it on a different branch. Leave the comment unresolved.

Reply:
Ok, leaving it open. I linked to this comment inside of the code.

Reply:
Covered in https://github.com/wasp-lang/wasp/issues/1846